test/component/response.spec.js

```diff
@@ -0,0 +1,373 @@
+const pactum = require('../../src/index');
+const { request, response } = pactum;
+const config = require('../../src/config');
+const { expect } = require('chai');
+
+describe('Response', () => {
+
+  it('with default expected response status - string value', async () => {
```

Comment:
Move negative scenarios related invalid usage to - test/unit/response.spec.js

Reply:
Updated the unit and component tests. Some of the negative cases in component tests were already covered in unit tests.
Moved the remaining to `test/unit/response.spec.js`